grc/scripts/gnuradio-companion

```diff
@@ -24,13 +24,14 @@ import warnings
 GR_IMPORT_ERROR_MESSAGE = """\
 Cannot import gnuradio.
 
-Is the model path environment variable set correctly?
+Is the Python path environment variable set correctly?
     All OS: PYTHONPATH
 
 Is the library path environment variable set correctly?
-    Linux: LD_LIBRARY_PATH
+    Linux and MacOSX: LD_LIBRARY_PATH
```

Comment:
delete "MacOSX" ... it doesn't use LD_LIBRARY_PATH ... and setting the DYLD_LIBRARY_PATH is dangerous unless you -really- know what you're doing!

Reply:
same here as in "grcc" below

Reply:
Per our discussion last night, please add something like: “On MacOSX / macOS the library search path is set primarily by the environment variable `DYLD_LIBRARY_PATH`. Because of the way the OSX dynamic library loader works, this variable works differently than on Linux. Its primary purpose is in finding libraries in a local path, such as when testing software before installing it or inside an application. We strongly recommend against setting this variable globally.”

Reply:
added https://wiki.gnuradio.org/index.php/ModuleNotFoundError#MacOSX_.2F_macOS